### DISCUSSION TITLE
Use markdown in fieldsets

### DISCUSSION THREAD

**Discussion by oke-oketech** [lucky-media/landtamic#34] [Q&A] [answered]
Hi there, I changed the text field with the handle description to a markdown field. In the editor everything looks fine and for example changing text to bold text works like a charm, but when I want to use a unsorted list the changes are not shown on my site. Do I need to change some other settings, to make this work? I am using markdown to make quick and simple edits to my texts.

**Comment by lokmanm** [comment, marked as answer]
Hey @oke-oketech,

If you use Markdown that will spit out HTML which is usually unstyled. To get that HTML styled you can check how we style the `text` coming from Bard ( which is HTML ) [here](https://github.com/lucky-media/landtamic/blob/main/resources/views/sets/blog/text.antlers.html).

It's as simple as wrapping your component with:

```antlers
<div
    class="max-w-3xl mx-auto prose prose-lg prose-p:text-gray-900 prose-a:text-secondary-900 prose-headings:text-primary-900">
    {{ markdown_handle_here }}
</div>
```

Change `markdown_handle_here` to your actual Markdown handle and it should work.

**Reply by oke-oketech** [reply]
Works! Thanks a lot!
